migrate/migrations/000007_add-posts-table.up.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE posts (
+    id INT UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+
+    type TINYINT UNSIGNED NOT NULL CHECK (type IN (0,1,2,3,4)),
+    audience TINYINT UNSIGNED NOT NULL CHECK (audience IN (0,1,2)),
```

Comment:
This is to store the audience of the post - currently thinking public, private & followers-only

Reply:
How would this affect the current feed queries?

Reply:
At the moment it wouldn't at all - we'd query everything - but eventually we could filter based on audience